src/GitVersionExe/IExecCommand.cs

```diff
@@ -0,0 +1,7 @@
+namespace GitVersion
+{
+    public interface IExecCommand
+    {
+        void Compute();
```

Comment:
I don't understand the relationship between `IExecCommand` and the `Compute` method. Would `Execute` be a more fitting method name or does it actually only perform computation? If so, why is the interface called `IExecCommand` and not `IComputeCommand`?

Reply:
Changed to `Execute` as it will execute msbuild or any external app

Reply:
I changed to `Compute` From `Execute` https://github.com/GitTools/GitVersion/pull/1865#discussion_r337246343

Reply:
I see. Sorry for the back and forth, I was perhaps not paying enough attention to what the method did in when I suggested the change from `Execute` to `Compute`. Since the method actually invokes `ExecuteCore.ExecuteGitVersion()` and the name of the interface is `IExecCommand`, `Execute()` makes most sense. Again, sorry for this confusion.

Reply:
That's fine, naming is always hard :)